client/payment-details/summary/test/index.tsx

```diff
@@ -207,92 +212,108 @@ describe( 'PaymentDetailsSummary', () => {
-		mockUseAuthorization.mockReturnValueOnce( {
-			authorization: {
-				captured: false,
-				charge_id: 'ch_mock',
-				amount: 1000,
-				currency: 'usd',
-				created: '2019-09-19 17:24:00',
-				order_id: 123,
-				risk_level: 1,
-				customer_country: 'US',
-				customer_email: 'test@example.com',
-				customer_name: 'Test Customer',
-				payment_intent_id: 'pi_mock',
-			},
-			isLoading: false,
-			isRequesting: false,
-			doCaptureAuthorization: jest.fn(),
-			doCancelAuthorization: jest.fn(),
+	describe( 'capture notification and fraud buttons', () => {
+		beforeAll( () => {
+			// Mock current date and time to fixed value in moment
+			const fixedCurrentDate = new Date( '2023-01-01T01:00:00.000Z' );
+			jest.spyOn( Date, 'now' ).mockImplementation( () =>
+				fixedCurrentDate.getTime()
+			);
 		} );
-		const charge = getBaseCharge();
-		charge.captured = false;
-
-		const container = renderCharge( charge );
-
-		expect(
-			screen.getByRole( 'button', { name: /Capture/i } )
-		).toBeInTheDocument();
-
-		expect(
-			screen.getByText( /You need to capture this charge/i )
-		).toHaveTextContent(
-			'You need to capture this charge before Sep 26, 2019 / 5:24PM'
-		);
-
-		expect( container ).toMatchSnapshot();
-	} );
 
-	test( 'renders the fraud outcome buttons', () => {
-		global.wcpaySettings.isFraudProtectionSettingsEnabled = true;
-
-		mockUseAuthorization.mockReturnValueOnce( {
-			authorization: {
-				captured: false,
-				charge_id: 'ch_mock',
-				amount: 1000,
-				currency: 'usd',
-				created: '2019-09-19 17:24:00',
-				order_id: 123,
-				risk_level: 1,
-				customer_country: 'US',
-				customer_email: 'test@example.com',
-				customer_name: 'Test Customer',
-				payment_intent_id: 'pi_mock',
-			},
-			isLoading: false,
-			isRequesting: false,
-			doCaptureAuthorization: jest.fn(),
-			doCancelAuthorization: jest.fn(),
+		afterAll( () => {
+			jest.spyOn( Date, 'now' ).mockRestore();
 		} );
-		const charge = getBaseCharge();
-		charge.captured = false;
 
-		const container = renderCharge( charge, {}, false, {
-			paymentIntent: paymentIntentMock,
-			fraudOutcome: latestFraudOutcomeMock,
+		test( 'renders capture section correctly', () => {
+			mockUseAuthorization.mockReturnValueOnce( {
+				authorization: {
+					captured: false,
+					charge_id: 'ch_mock',
+					amount: 1000,
+					currency: 'usd',
+					created: moment.utc().format(),
+					order_id: 123,
+					risk_level: 1,
+					customer_country: 'US',
+					customer_email: 'test@example.com',
+					customer_name: 'Test Customer',
+					payment_intent_id: 'pi_mock',
+				},
+				isLoading: false,
+				isRequesting: false,
+				doCaptureAuthorization: jest.fn(),
+				doCancelAuthorization: jest.fn(),
+			} );
+			const charge = getBaseCharge();
+			charge.captured = false;
+
+			const container = renderCharge( charge );
+
+			expect(
+				screen.getByRole( 'button', { name: /Capture/i } )
+			).toBeInTheDocument();
+
+			expect(
```

Comment:
I'd say that this assertion seems to be duplicated with the` toMatchSnapshot()` below.

Reply:
It was indeed redundant! I have retained the snapshot assertion and removed the exact element compare.